benchmarks/src/main/reports/SetBenchmarks.txt

```diff
@@ -0,0 +1,21 @@
+# JMH version: 1.25
```

Comment:
I would remove this file and publish the result via pr comment, as discussed over chat.